### PR TITLE
Tools: Merged improved TS lib and Dashboards API.

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Review by jomar-honsi — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]
Docs are generated correctly

File: tools/api-docs/api-classes.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 import FS from 'node:fs/promises';
 import FSLib from '../libs/fs.js';
-import TSLib from '../libs/ts.js';
+import TSLib from '../libs/TS.js';
```
Comment (KacperMadej) [inline tools/api-docs/api-classes.ts:20]: `Error: Cannot find module '../libs/TS.js'`. This and the one in api-options should be changed from .js to no extension (`import TSLib from '../libs/TS';`).

**Review by KacperMadej — APPROVED** [review]
When the one thing from my comment is fixed then it works. Other then that LGTM 👍

**Review by sebastianbochan — APPROVED** [review]

**Review by ddragula — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**